tests/doc_scan_service/session/retrieve/custom.account.watchlist.ca.search.config.response.spec.js

```diff
@@ -0,0 +1,166 @@
+      });
+
+      it('#getTags', () => {
+        expect(customAccountWatchlistCaSearchConfigResponse.getTags()).toStrictEqual({});
+      });
+
+      it('#getClientRef', () => {
+        expect(customAccountWatchlistCaSearchConfigResponse.getClientRef()).toBe('an-client-ref');
+      });
+    });
+    describe('with non empty tags in response body', () => {
+      beforeEach(() => {
+        const response = Object.assign({}, minimumResponseBody, {
+          tags: {
+            nickname: 'Jack the Ripper',
+            classification: 'dangerous',
+          },
+        });
+        // eslint-disable-next-line max-len
+        customAccountWatchlistCaSearchConfigResponse = new CustomAccountWatchlistCaSearchConfigResponse(response);
+      });
+
+      it('#getTags', () => {
+        expect(customAccountWatchlistCaSearchConfigResponse.getTags()).toStrictEqual({
+          nickname: 'Jack the Ripper',
+          classification: 'dangerous',
+        });
+      });
+    });
+    describe('with sources in response body', () => {
+      describe('of type PROFILE', () => {
+        beforeEach(() => {
+        // eslint-disable-next-line max-len
+          const response = Object.assign({}, minimumResponseBody, {
+            sources: {
+              type: 'PROFILE',
+              search_profile: 'the-profile-being-search-for',
+            },
+          });
+          // eslint-disable-next-line max-len
+          customAccountWatchlistCaSearchConfigResponse = new CustomAccountWatchlistCaSearchConfigResponse(response);
+        });
+
+        it('#getSources', () => {
+          const sources = customAccountWatchlistCaSearchConfigResponse.getSources();
+          expect(sources).toBeInstanceOf(SearchProfileSourcesResponse);
+          expect(sources.getType()).toBe('PROFILE');
+          expect(sources.getSearchProfile()).toBe('the-profile-being-search-for');
+        });
+      });
+      describe('of type TYPE_LIST', () => {
+        beforeEach(() => {
+        // eslint-disable-next-line max-len
+          const response = Object.assign({}, minimumResponseBody, {
+            sources: {
+              type: 'TYPE_LIST',
+              types: ['type1', 'type2'],
+            },
+          });
+          // eslint-disable-next-line max-len
+          customAccountWatchlistCaSearchConfigResponse = new CustomAccountWatchlistCaSearchConfigResponse(response);
+        });
+
+        it('#getSources', () => {
+          const sources = customAccountWatchlistCaSearchConfigResponse.getSources();
+          expect(sources).toBeInstanceOf(TypeListSourcesResponse);
+          expect(sources.getType()).toBe('TYPE_LIST');
+          expect(sources.getTypes()).toStrictEqual(['type1', 'type2']);
+        });
+      });
+    });
+    describe('with matching strategy in response body', () => {
+      describe('of type EXACT', () => {
+        beforeEach(() => {
+        // eslint-disable-next-line max-len
```

Comment:
Guessing we don't need this any more
```suggestion
```

Reply:
👍 yeah, there were a couple of those, annoying as they stick around after refactoring, it's almost that you'd need a rule in eslint to warn that you are using a non necessary `eslint-disable`...